dev_requirements.txt

```diff
@@ -4,4 +4,9 @@ flake8
 black
 mypy
 sphinx
-myst-nb
\ No newline at end of file
+myst-nb
+quimb
+opt_einsum
+autoray
+furo==2021.11.16
```

Comment:
I think these should be installed with `pip install quimb[tensor]`. I was able to run this example with just that.

Reply:
Will check in future PR